### PR TITLE
Dev fix error

### PR DESCRIPTION
**Pull request by Jinpeiqi**
@rong4188 @Mooophy

### CODE REVIEW

File: MeetU/MeetU/Scripts/Views/Profile/profile-img-edit.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿<div ng-controller="ProfileImageEditController as ProfileImageEdit">
+    <subpage-nav hassubmit="true" on-submit="ProfileImageEdit.submit()" title="'Edit Meetup'"></subpage-nav>
+        <div>
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/Views/Profile/profile-img-edit.html]: indentation is not correct here

File: MeetU/MeetU/Scripts/Controllers/Profile/profile-img-edit.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                alert("Please select your image");
+                return;
+            }
+            if ((/^data:image\/png;base64/.test($scope.CroppedImage) | /^data:image\/jpeg;base64/.test($scope.CroppedImage))==0) {
+                alert("Image must be png/jpeg format");
+                return;
+            }
+            //console.log(vm.loggedUser.userId); test
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/Controllers/Profile/profile-img-edit.js]: please remove comment code like this

File: MeetU/MeetU/Scripts/Controllers/Profile/profile-img-edit.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    ProfileImageEditController.$inject = ["$scope", "$resource", "$location"];
+    function ProfileImageEditController($scope, $resource, $location) {
+        var vm = this;
+        var User = $resource('/api/loggedUser');
+        var ProFilePicture = $resource('api/ProfilePicture');
+
+        User.query(function (user) {
+            vm.loggedUser = user[0];
+        }).$promise.then(function () {
+            //console.log(vm.loggedUser); test
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/Controllers/Profile/profile-img-edit.js]: ???

File: MeetU/MeetU/Scripts/Controllers/Profile/profile-img-edit.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        })
+
+        $scope.Image = '';
+        $scope.CroppedImage = '';
+        var handleFileSelect = function (evt) {
+            var file = evt.currentTarget.files[0];
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/Controllers/Profile/profile-img-edit.js]: DO NOT USE ABBREVATION

**Comment by Jinpeiqi** [comment]
@rong4188

File: MeetU/MeetU/Scripts/Controllers/Profile/profile-img-edit.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+            var file = event.currentTarget.files[0];
+            var reader = new FileReader();
+            reader.onload = function (event) {
+                $scope.$apply(function ($scope) {
+                    $scope.Image = event.target.result;
+                });
+            };
+            reader.readAsDataURL(file);
+        };
+        angular.element(document.querySelector('#fileInput')).on('change', handleFileSelect);
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/Controllers/Profile/profile-img-edit.js]: should change #fileinput to a class with prefix. we should avoid a common used name like this.

File: MeetU/MeetU/Scripts/Views/Profile/profile-img-edit.html [hunk 6]
```diff
@@ -0,0 +1,11 @@
+﻿<div ng-controller="ProfileImageEditController as ProfileImageEdit">
+    <subpage-nav hassubmit="true" on-submit="ProfileImageEdit.submit()" title="'Edit Image'"></subpage-nav>
+        <div>
+            <div>Select an image file: <input type="file" id="fileInput" /></div>
+            <div class="crop-area">
+                <img-crop image="Image" result-image="CroppedImage" result-image-size="140"></img-crop>
+            </div>
+            <div>Cropped Image:</div>
+            <div><img name="CroppedImage" ng-src="{{CroppedImage}}" /></div>
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/Views/Profile/profile-img-edit.html]: where is 'name' used?

**Comment by Jinpeiqi** [comment]
@rong4188

**Pull request merged**